### ISSUE TITLE
Update replicated group, respect orientation

### ISSUE THREAD

**Issue by shufps** [MitjaNemec/ReplicateLayout#75]
What is really missing is to update previously replicated groups and respect the group orientation.

It should update the group without error message and the group and the parts inside of the group should keep it's orientation.

**Comment by MitjaNemec** [comment]
Yeah the grouping feature is quite lacking robustness and features. But for the time being, I am lacking time and motivation to do anything about it. Moreover so as python API is going to be removed in the future.

Thanks for understanding

**Comment by shufps** [comment]
> Moreover so as python API is going to be removed in the future.

Wait what? :flushed:  Why would they do this? I would never ever again update KiCad then ...
